recipes/thorvg/all/conanfile.py

```diff
@@ -200,3 +202,15 @@ def package_info(self):
             self.cpp_info.defines = ["TVG_STATIC"]
         else:
             self.cpp_info.defines = ["TVG_EXPORT", "TVG_BUILD"]
+
+        if self.options.with_threads and Version(self.version) >= "0.15.0":
+            openmp_flags = []
+            if is_msvc(self):
+                openmp_flags = ["-openmp"]
+            elif self.settings.compiler == "gcc":
+                openmp_flags = ["-fopenmp"]
+            elif self.settings.compiler in ["clang", "apple-clang"]:
+                openmp_flags = ["-Xpreprocessor", "-fopenmp"]
+            self.cpp_info.cxxflags.extend(openmp_flags)
+            self.cpp_info.exelinkflags.extend(openmp_flags)
+            self.cpp_info.sharedlinkflags.extend(openmp_flags)
```

Comment:
`llvm-openmp` recipe is already setting some flags in the cpp_info. I think that we can remove all of those:

```suggestion
```

Have you checked it locally?

Reply:
@franramirez688 
Thank you for your review!
I try to remove all codes, but I met compilation errors:
```
[100%] Linking CXX executable test_package
/usr/bin/ld: lib/libthorvg.a(renderer_sw_engine_tvgSwRenderer.cpp.o): in function `tvg::SwRenderer::init()':
tvgSwRenderer.cpp:(.text+0x217c): undefined reference to `omp_set_num_threads'
/usr/bin/ld: lib/libthorvg.a(renderer_sw_engine_tvgSwRaster.cpp.o): in function `rasterXYFlip(unsigned int*, unsigned int*, int, int, int, SwBBox const&, bool) [clone ._omp_fn.0]':
tvgSwRaster.cpp:(.text+0x244d): undefined reference to `omp_get_num_threads'
/usr/bin/ld: tvgSwRaster.cpp:(.text+0x2454): undefined reference to `omp_get_thread_num'
/usr/bin/ld: lib/libthorvg.a(renderer_sw_engine_tvgSwRaster.cpp.o): in function `rasterXYFlip(unsigned int*, unsigned int*, int, int, int, SwBBox const&, bool)':
tvgSwRaster.cpp:(.text+0xdefd): undefined reference to `GOMP_parallel'
/usr/bin/ld: lib/libthorvg.a(renderer_sw_engine_tvgSwPostEffect.cpp.o): in function `_gaussianBlur(unsigned char*, unsigned char*, int, int, int, SwBBox const&, int, int, bool) [clone ._omp_fn.0]':
tvgSwPostEffect.cpp:(.text+0x15): undefined reference to `omp_get_num_threads'
/usr/bin/ld: tvgSwPostEffect.cpp:(.text+0x1c): undefined reference to `omp_get_thread_num'
/usr/bin/ld: lib/libthorvg.a(renderer_sw_engine_tvgSwPostEffect.cpp.o): in function `effectGaussianBlur(SwImage&, SwImage&, SwBBox const&, tvg::RenderEffectGaussian const*)':
tvgSwPostEffect.cpp:(.text+0x1094): undefined reference to `GOMP_parallel'
/usr/bin/ld: tvgSwPostEffect.cpp:(.text+0x1152): undefined reference to `GOMP_parallel'
/usr/bin/ld: tvgSwPostEffect.cpp:(.text+0x1258): undefined reference to `GOMP_parallel'
/usr/bin/ld: tvgSwPostEffect.cpp:(.text+0x1412): undefined reference to `GOMP_parallel'
/usr/bin/ld: tvgSwPostEffect.cpp:(.text+0x1518): undefined reference to `GOMP_parallel'
/usr/bin/ld: lib/libthorvg.a(renderer_sw_engine_tvgSwPostEffect.cpp.o):tvgSwPostEffect.cpp:(.text+0x15df): more undefined references to `GOMP_parallel' follow
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/test_package.dir/build.make:107: test_package] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/test_package.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
```

It is caused by the lack of `execlinkflags`.
So I am going to modify this recipe as followings:
```python
if self.options.with_threads and Version(self.version) >= "0.15.0":
    openmp_flags = []
    if is_msvc(self):
        openmp_flags = ["-openmp"]
    elif self.settings.compiler == "gcc":
        openmp_flags = ["-fopenmp"]
    elif self.settings.compiler in ["clang", "apple-clang"]:
        openmp_flags = ["-Xpreprocessor", "-fopenmp"]
    self.cpp_info.exelinkflags.extend(openmp_flags)
```